bundle/bundle.go

```diff
@@ -249,3 +249,15 @@ func (b *Bundle) AuthEnv() (map[string]string, error) {
 	cfg := b.client.Config
 	return auth.Env(cfg), nil
 }
+
+func (b *Bundle) NormalizePath(path string) (string, error) {
+	var err error
+	if filepath.IsAbs(path) {
```

Comment:
Not entirely sure about making absolute paths relative ones. Some of the code (like UploadFile for artifacts or some dashboard code later) relies on such paths to be absolute